local-cli/server/server.js

```diff
@@ -12,6 +12,7 @@ const chalk = require('chalk');
 const formatBanner = require('./formatBanner');
 const path = require('path');
 const runServer = require('./runServer');
+const findSymlinksPaths = require('./findSymlinksPaths');
```

Comment:
What I mean is - you might actually want to call findSymlinkPaths to... you know... find the symlink paths and perhaps append them where they belong, no? :)